### ISSUE TITLE
standard_title() displays incorrect McIlwain L when plotting

### ISSUE THREAD

**Issue by joshuarwood** [USRA-STI/gdt-fermi#16]
Using standard_title() displays incorrect McIlwain L when plotting. Here is the code to reproduce the issue for burst number 231003905:
```
import matplotlib.pyplot as plt

from gdt.missions.fermi.time import Time
from gdt.missions.fermi.plot import FermiEarthPlot
from gdt.missions.fermi.gbm.finders import TriggerFtp
from gdt.missions.fermi.gbm.trigdat import Trigdat

# download trigdat file
bn = "231003905"
TriggerFtp(bn).get_trigdat(".")

trigdat = Trigdat.open("glg_trigdat_all_bn231003905_v01.fit")
trigtime = Time(trigdat.trigtime, format='fermi')

mcilwain_plot = FermiEarthPlot(mcilwain=True)
mcilwain_plot.add_spacecraft_frame(trigdat.poshist, trigtime=trigtime)
mcilwain_plot.standard_title()

plt.gcf().set_figheight(5)
plt.show()
```
In this case, the McIlwain L should be 1.56 but instead the title reports a value of 1.09. 1.09 is clearly inconsistent with the color code corresponding to the spacecraft position (see attached screenshot). 
<img width="1165" alt="Screenshot 2023-11-20 at 2 17 02 PM" src="https://github.com/USRA-STI/gdt-fermi/assets/26443181/18f21024-4c77-409a-b3eb-fd6ad9763fe1">

The source of the issue is the fact that standard_plot() does not account for East versus West when getting longitude from the coordinates of the Fermi spacecraft icon on line 344 of src/gdt/missions/fermi/plot.py:
```
if self.spacecraft is not None:
    coord = self.spacecraft.coordinates
    title = 'Latitude, East Longitude: ({0}, {1})\n'.format(*coord)
    mcl = calc_mcilwain_l(float(coord[0][:-1]), float(coord[1][:-1])) # this line is the issue. coord[1] corresponds to "99.70W"                                                                                                                      
                                                                      # and using coord[1][:-1] removes the "W" before float conversion
```
A potential fix is:
```
if self.spacecraft is not None:
    coord = self.spacecraft.coordinates
    title = 'Latitude, East Longitude: ({0}, {1})\n'.format(*coord)
    lat = float(coord[0][:-1])
    lon = float(coord[1][:-1]) * (-1 if "W" in coord[1] else 1)
    mcl = calc_mcilwain_l(lat, lon)
```

**Comment by joshuarwood** [comment]
I just noticed that lat also needs to be updated to account for N/S.
```
lat = float(coord[0][:-1]) * (-1 if "S" in coord[0] else 1)
```

**Comment by AdamGoldstein-USRA** [comment]
This has been resolved with commit 565914e.

**Comment by AdamGoldstein-USRA** [comment]
This has been resolved with commit 565914e.
